src/views/components/RecordingAndRTMPForm.jsx

```diff
@@ -54,10 +57,39 @@ export const RecordingAndRTMPForm = ({
             />
           </div>
         </div>
+
+        <div className={defaultClasses.formInner}>
```

Comment:
The form component is getting quite big. Let's break it down by creating sub components in this file